include/zephyr/kernel/internal/sched_priq.h

```diff
@@ -51,8 +63,8 @@ struct k_thread *z_priq_rb_best(struct _priq_rb *pq);
  * to represent their requirements.
  */
 struct _priq_mq {
-	sys_dlist_t queues[32];
-	unsigned int bitmask; /* bit 1<<i set if queues[i] is non-empty */
+	sys_dlist_t queues[PRI_QUEUE_BITMAP_SIZE * BIT_NUM];
+	unsigned int bitmask[PRI_QUEUE_BITMAP_SIZE]; /* bit 1<<i set if queues[i] is non-empty */
```

Comment:
You don't need the full bitmap size, just the number of actual priorities.  Also the comment is now confusing as it doesn't specify which array element comes first.